.gitignore

```diff
@@ -12,3 +12,4 @@ __pycache__
 *.swp
 .idea
 *.iml
+.history
```

Comment:
Typically pull requests for recipes don't make changes outside of `/recipes/<pkg>`. What is `.history` for?

Reply:
I'll undo it.

Reply:
Fixed.